e2e/imagestore/imagestore_test.go

```diff
@@ -90,16 +77,31 @@ func TestImageStore(t *testing.T) {
 	require.NoError(ct.Kubeclient.WaitForStatefulSet(ctx, ct.Namespace, "coordinator"))
 
 	for name, tc := range tests {
-		if platforms.IsGPU(platform) != tc.gpu {
-			continue
-		}
-
 		t.Run(name, func(_ *testing.T) {
 			require.NoError(ct.Kubeclient.WaitForPod(ctx, ct.Namespace, tc.name))
 
 			pod, err := ct.Kubeclient.Client.CoreV1().Pods(ct.Namespace).Get(ctx, tc.name, metav1.GetOptions{})
 			require.NoError(err)
 
+			var expectedKibiBytes int
+			switch tc.annotation {
+			case "0":
+				defaultMemory := platforms.DefaultMemoryInMebiBytes(platform) * 1024 * 1024
+				for _, c := range pod.Spec.Containers {
+					memory := c.Resources.Limits[corev1.ResourceMemory]
+					defaultMemory += int(memory.Value())
+				}
+				// When the imagestore is disabled, the expectedKibiBytes are (the total VM memory in KiB - internal VM overhead) / 2
+				// For more information on the internal VM overhead, see https://github.com/edgelesssys/contrast/pull/1196
+				expectedKibiBytes = ((defaultMemory / 1024) - 99) / 2
```

Comment:
This assumes a  _99kiB_ overhead, right? Shouldn't we use something closer to 200MiB?

Reply:
Note that the default memory in that table was 256MiB, but now is 512MiB, so I was looking at the rows near 1G total VM memory.